Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DPEComponentAdapter.h

```diff
@@ -46,8 +46,6 @@ namespace AZ::DocumentPropertyEditor
         //! Trigger a refresh based on messages from the listeners
         void DoRefresh();
 
-        Dom::Value HandleMessage(const AdapterMessage& message) override;
-
```

Comment:
The code in that function was added so that prefab propagation continues to happen even with dpe based inspector.

The idea was that we'd mark the owning entity dirty and let prefab system calculate the patches itself . But now, we are creating the prefab patches from DPE patches. So that whole logic is no longer needed.

And I cannot think of a reason why that'd be necessary in non-prefab based component editors like lyshine as they don't use prefabs. So, removing it for now made most sense.

@amzn-tmryan , let me know if you still find any reason for having this around.

Reply:
> The idea was that we'd mark the owning entity dirty and let prefab system calculate the patches itself . But now, we are creating the prefab patches from DPE patches. So that whole logic is no longer needed.

Yeah I understand this. The thing I don't follow is (after property edit) how the new logic gets executed in the case where DPE=on & override=off. ComponentEditor only knows DPEComponentAdapter and it doesn't know the new logic I think.

In your current version, if you turn off the inspector override flag, is property edit still working as expected with undo/redo?

Reply:
Good point. I hadn't thought about that. keeping the possibility of launching DPE for inspector first without overrides, I'll add back this deleted code. Once I add it back, propagation works as usual.

Reply:
> Once I add it back, propagation works as usual.

Would the old code interfere with the new `UpdateDomContents` code in PrefabComponentAdapter when override flag is on? I think the time `UpdateDomContents` is executed, it would also trigger `OnChanged` message then the ComponentAdapter would handle it.

If yes, we would need to add skip message handling there if override flag is on.